test/integration/roles/setup_postgresql_db/tasks/main.yml

```diff
@@ -8,24 +8,10 @@
       paths: '../vars'
 
 # Make sure we start fresh
-- name: remove rpm dependencies for postgresql test
-  package: name={{ postgresql_package_item }} state=absent
-  with_items: "{{postgresql_packages}}"
-  loop_control:
-    loop_var: postgresql_package_item
-  when: ansible_os_family == "RedHat"
-
-- name: remove dpkg dependencies for postgresql test
-  apt: name={{ postgresql_package_item }} state=absent
-  with_items: "{{postgresql_packages}}"
-  loop_control:
-    loop_var: postgresql_package_item
-  when: ansible_pkg_mgr  ==  'apt'
-
 - name: remove old db (red hat)
```

Comment:
I've previously spoken with abadger regarding this. Without this runs of tests can be effected by the existing state of the system under tests.  

Although this isn't an issue for Travis runs (as it's a fresh docker everytime) it does effect people testing in there own systems outside of docker.